ansible/templates/aeternity.backup.yaml

```diff
@@ -2,8 +2,8 @@
 chain:
   persist: true
   db_path: "./db{{ db_version|mandatory }}"
-{% if node_config['fork_management'] %}
-{% if node_config['fork_management']['network_id'] %}
+{% if node_config['fork_management'] is defined %}
+{% if node_config['fork_management']['network_id'] is defined %}
 fork_management:
     network_id: {{ node_config.fork_management.network_id }}
```

Comment:
it should be safe to use `ae_mainnet` as default instead of the ifs

https://github.com/aeternity/aeternity/blob/4fecd35b2b3bf2b311a306d13db22fb20864c496/apps/aecore/src/aec_governance.erl#L229
https://github.com/aeternity/aeternity/blob/4fecd35b2b3bf2b311a306d13db22fb20864c496/apps/aecore/src/aec_governance.erl#L38

Reply:
should or it's safe? and i can use network_id as default for fork_management.network_id but still will fail on fork_management do not exist.